src/views/Vendor/Vendor.jsx

```
@@ -198,272 +243,312 @@ const Vendor = () => {
+                  onChangeRowsPerPage={onChangeRowsPerPage}
+                  onChangePage={onChangePage}
+                />
+              </>
+            )}
+          </>
+        ) : (
+          <div className="card-view">
+            {vendors.map((item) => {
+              return (
+                <>
+                  <div className="cursor-pointer card bg-white border-none" style={{ filter: "drop-shadow(0px 4px 3px rgba(0, 0, 0, 0.07)) drop-shadow(0px 2px 2px rgba(0, 0, 0, 0.06))" }}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)